releasenotes/notes/fix-pythonpath-3e26d996b9df139a.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixed layering of package installations by tracking the parent-child
```

Comment:
Since the change to using PYTHONPATH hasn't actually been released yet I don't think we need the release note. Although this snippet would be great in the commit message as it succinctly describes the fix here :+1:

Reply:
Great to hear that's working well! 😅  I've removed the relnote and moved the message to the commit 👍